### ISSUE TITLE
Luciferous Woods (fixed)

### ISSUE THREAD

**Issue by enochmeow** [PFCKrutonium/OpenRCT2-OpenScenarios#4]
here you go! 
[Luciferous Woods.zip](https://github.com/PFCKrutonium/OpenRCT2-OpenScenarios/files/1177686/Luciferous.Woods.zip)

**Comment by Krutonium** [comment]
Thanks! Going to go play-test this now.

**Comment by Krutonium** [comment]
So this is massively over difficult, I've failed to beat it even with someone else helping me.

**Comment by Keatzee** [comment]
I just completed it. It's not impossible, you just need to take an unorthodox approach. I chose to make tons of cheap little looping coasters right off the bat, charge $5 a ride and $3 for photos. Three months later, I barely escaped the interest and turned a profit. Then I sped up my game and slowly paid off the debt for 8 years. Finally I spent a few years spamming prefab coasters to build up my park value. 

To be honest, I feel like being rushed to escape interest, and waiting years to pay off a debt isn't really in the spirit of the original game. Also, a scenario, no matter how hard, should be achievable through conventional means--that is, you shouldn't have to exploit a quirk in the game mechanics to win.

So for those reasons, I think it needs a little bit of work. That said, it's still a really nice scenario and would make a good addition to the pack. I would suggest toning down the financial situation a bit and allow for more thinking outside-the-box. It's ultimately up to @PFCKrutonium , but that's just my opinion as a tester.

I hope I haven't discouraged you, and I look forward to trying the new version!
